### ISSUE TITLE
Not an issue, but a thank you

### ISSUE THREAD

**Issue by R-Teer** [profzei/Matebook-X-Pro-2018#93]
Hey @profzei 

I just wanted to extend a thank you for all of your efforts and hard work in getting this project as far as it has gotten over the past year. Can't thank you enough for your consistent support and guidance regarding this project. 

Also want to extend thanks to all those who have contributed to the mxp hackintosh project as well as those who have worked on development of Intel wifi support. 

I see you have been hard at work over the past few weeks rebuilding the patches for Big Sur and to ensure better support across the patches. 

Really looking forward to your latest update, if you require any assistance with anything just let me know! 

PS: Maybe it's time to set up a donate link 👍

All the best! 

R

**Comment by profzei** [comment]
**Thank you very much for your feedback!**

With latest release I'm pretty sure Catalina **is** very stable!
I only **hope** Big Sur is the same... even if I read about some issues in power management (in Big Sur with high cpu occupancy due to `kernelmanagerd`...) probably due to apps or extensions not yet fully compatible with BS!

PS: btw I don't even know how to setup a donate link! XD

**Comment by Vatri91** [comment]
@profzei You should sign up on PayPal. I live in a shitty country like Poland with shitty earnings, but I will gladly send you some PLN for Polish vodka ^^

**Comment by Vatri91** [comment]
@profzei Put your donation information on paypall at the top of your website. The coffee is flying to you. Thanks for your work.

**Comment by profzei** [comment]
@Vatri91 ...Thank you very much! I'm doing this for fun (but sometimes it's also very stressing... XD), but honestly I didn't expect so much appreciation! ;)

**Comment by wiregen** [comment]
I would also gladly donate. For keeping your github updated and actively responding here.

**Comment by mmenendezg** [comment]
@profzei I want to add my Thank you to this thread. Thanks to you I could really enjoy my MBXP with a great operative system, and as a geek enjoyed the process of setting the Hackintosh in my own laptop. This has been kinda frustrating getting to understand how the system works, but you have made an excellent job in your tutorial and explaining everything. 

You are great, keep rockin'!

**Comment by wiregen** [comment]
I just saw the donate button on your repo. Just shot you 5 Euro! Coffee's on me

**Comment by profzei** [comment]
@mmenendezg @wiregen Thank you so much for your appreciation!

**Comment by RoderickQiu** [comment]
Yes, thank you a lot @profzei as it's the only working solution for my MBXP 2018 to use Big Sur and it has been so fluent to use your EFI (I've been stuck for long to upgrade to Big Sur until I find that you've got an awesome macOS 11 solution!)

**Comment by myselfghost** [comment]
Thank you very much for your work.

**Comment by robvalca** [comment]
I've already expressed my gratitude in another post, but want also to say thank you here. I've never been happy with my Huawei. The device is really nice, but the experience on Windows/Linux was not very good (specially on windows). So my laptop was sitting in one corner getting dust... until I found this guide, and thanks to it, the laptop as become again my main computer and I'm enjoying it like never using Catalina. It's the closest experience to Mac that I've experimented!
Again, thanks !
Roberto

**Comment by Vatri91** [comment]
I'm testing new versions and tweaks of your work right now, and everything works perfectly. WOW. The project evolved from curiosity to a fully functioning system that I use on a daily basis. I don't even have Windows anymore.

Thank you so much

**Comment by profzei** [comment]
@Vatri91 Thank you very much for your appreciation!
